### PR TITLE
Add team handbook bug template

### PR DESCRIPTION
**Pull request by jverkoey**
Add a team handbook bug template for filing team handbook change requests.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/Team_handbook.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+Cons:
+
+- Individual items describing the negative aspects of this variant.
+
+Action items:
+
+- A list of actions that will have to be taken in order to make this handbook change.
+
+## Variant 2: A single-sentence description of this variant.
```
Comment (andrewoverton) [inline .github/ISSUE_TEMPLATE/Team_handbook.md]: What is the difference between the two variants? It looked like there was one but now there's not

Comment (jverkoey) [reply .github/ISSUE_TEMPLATE/Team_handbook.md]: Ah, hm. This is just meant to be an example. Let me try again...

Comment (jverkoey) [reply .github/ISSUE_TEMPLATE/Team_handbook.md]: Updated.

File: .github/ISSUE_TEMPLATE/Team_handbook.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+---
+name: Team handbook
+about: Propose a change to the team's handbook.
+
+---
+
+- [ ] The issue's title has a prefix of `[Team handbook]`.
+- [ ] The issue has been labeled with `Team handbook`.
+- [ ] There is more than one variant in the proposal.
```
Comment (ianegordon) [inline .github/ISSUE_TEMPLATE/Team_handbook.md]: For new concepts this makes sense.
For changes / modifications to existing concepts, perhaps less so.

**Review by ianegordon — APPROVED** [review]
Consider comments before merge.

**Review by andrewoverton — APPROVED** [review]
LGTM, no strong feelings

File: .github/ISSUE_TEMPLATE/Team_handbook.md (hunk 2 above)
Comment (jverkoey) [reply .github/ISSUE_TEMPLATE/Team_handbook.md]: Dropped the multiple variants requirement.
